app/services/doc_auth/socure/requests/document_request.rb

```diff
@@ -47,6 +47,43 @@ def handle_http_response(http_response)
           JSON.parse(http_response.body, symbolize_names: true)
         end
 
+        def handle_invalid_response(http_response)
+          message = [
+            self.class.name,
+            'Unexpected HTTP response',
+            http_response.status,
+          ].join(' ')
+          exception = DocAuth::RequestError.new(message, http_response.status)
+
+          response_body = begin
+            http_response.body.present? ? JSON.parse(http_response.body) : {}
+          rescue JSON::JSONError
+            {}
+          end
+
+          handle_connection_error(
+            exception: exception,
+            status_code: response_body.dig('status', 'code'),
+            status_message: response_body.dig('status', 'message'),
+          )
+        end
+
+        def handle_connection_error(exception:, status_code: nil, status_message: nil)
+          NewRelic::Agent.notice_error(exception)
+          {
+            success: false,
+            errors: { network: true },
+            exception: exception,
+            extra: {
+              vendor: 'Socure',
+              selfie_live: false,
+              selfie_quality_good: false,
```

Comment:
```suggestion
```
we're not implementing selfie and this is not a doc auth response ... we could eliminate these